src/main/java/seedu/address/logic/commands/ListFavCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+import seedu.address.model.person.IsFavouritePredicate;
+
+/**
+ * Finds and lists all persons in ModuLink whose Student ID contains any of the argument keywords.
```

Comment:
Need to update JavaDocs to 'Finds and lists all persons in ModuLink who are marked as favourites' or something